src/MuTalk-Model/MTAnalysis.class.st

```diff
@@ -250,8 +265,8 @@ MTAnalysis >> run [
 	We obtain a result for each mutant generated"
 
 	^ [
-	  budget start.
 	  self initialTestRun.
+	  self startBudget.
```

Comment:
Yes, this is what we talked about.
We should explain why this in a comment so we don't forget in the future :).

Reply:
Will do it